cli/azd/pkg/project/scaffold_gen_environment_variables.go

```diff
@@ -579,7 +579,7 @@ func GetResourceConnectionEnvs(usedResource *ResourceConfig,
 		return []scaffold.Env{
 			{
 				Name:  "spring.config.import",
-				Value: fmt.Sprintf("optional:configserver:%s", scaffold.GetContainerAppHost(usedResource.Name)),
+				Value: fmt.Sprintf("optional:configserver:%s?fail-fast=true", scaffold.GetContainerAppHost(usedResource.Name)),
```

Comment:
Make it restart to reconcile if config server unavailable, until fetch from config server. This can avoid the application starts up without remote config server.